packages/contracts/contracts/minter-suite/Minters/MinterSetPriceERC20V4.sol

```diff
@@ -361,6 +443,25 @@ contract MinterSetPriceERC20V4 is
     function purchaseTo_do6(
         address _to,
         uint256 _projectId
+    ) external payable returns (uint256 tokenId) {
+        // pass max price as msg.value, currency address as ETH
+        return
+            purchaseTo_do6({
+                _to: _to,
+                _projectId: _projectId,
+                _maxPricePerToken: msg.value,
+                _currencyAddress: address(0)
+            });
+    }
+
+    /**
+     * @notice gas-optimized version of purchaseTo(address, uint256, uint256, address).
+     */
+    function purchaseTo_do6(
```

Comment:
I think can just make this `purchaseTo_bjU` to update the gas-optimized name for the new function signature.
Could also just remove all the gas-optimized function names if you want.
reference: https://emn178.github.io/solidity-optimize-name/

Reply:
Updated! 🙏